pallets/ethereum-transactions/src/lib.rs

```diff
@@ -452,6 +452,7 @@ impl<T: Config> Pallet<T> {
             EthTransactionType::PublishRoot(_) => Some(Self::get_publish_root_contract()),
             EthTransactionType::ActivateCollator(_) | EthTransactionType::DeregisterCollator(_) =>
                 Some(T::ValidatorManagerContractAddress::get()),
+            EthTransactionType::TriggerGrowth(_) => Some(Self::get_publish_root_contract()),
```

Comment:
is this correct? or will there be a followup pr?

Reply:
It will work for now but needs correcting once the single avn-bridge contract storage item changes go in. Should get picked up on that merge.